apps/web/components/Organization/SidebarCard/SidebarCard.tsx

```diff
@@ -0,0 +1,57 @@
+import {
+  Box,
+  Button,
+  GridColumn,
+  GridContainer,
+  GridRow,
+  Link,
+  Text,
+} from '@island.is/island-ui/core'
+import React from 'react'
+import { SidebarCard as Card } from '@island.is/web/graphql/schema'
+
+interface SidebarCardProps {
+  sidebarCard: Card
+}
+
+export const SidebarCard: React.FC<SidebarCardProps> = ({ sidebarCard }) => (
+  <Box marginTop={4} border="standard" borderRadius="large" padding={4}>
+    <GridContainer>
+      <GridRow>
+        <GridColumn span={['3/12', '3/12', '12/12']}>
+          <Box
+            display="flex"
+            justifyContent={'center'}
+            alignItems={'center'}
+            paddingBottom={3}
+          >
+            <img
+              src={
+                'https://images.ctfassets.net/8k0h54kbe6bj/2c9RXjtApgqkUWeKh0s1xP/820afa7b351e638473bff013277929ea/Vector.svg'
```

Comment:
This approach might be fragile since there is no indication this asset is in use inside Contentful.
Did you consider using the Icon component from island-ui?
It seems to already have a similar warning icon with a red variant:
https://ui.devland.is/?path=/docs/core-icon--all-icons